CONTRIBUTING.md

```diff
@@ -322,3 +322,20 @@ precompile/build-all-addons.js
 ```
 
 NOTE: It is recommended to comment out the ABI versions you don't want to generate/regenerate in the `build-all-addons` script. Alternatively, if you want to regenerate all builds that's fine too, but it takes longer.
+
+#### How to publish local layer to AWS
+
+To publich local layer to AWS with the AWS CLI from local, you need an AWS account and IAM credentials. When running AWS CLI commands, the AWS CLI needs to have access to those AWS credentials.
+jq and Docker must be installed as prerequisites.
```

Comment:
These requirements are all listed in line 3
https://github.com/instana/nodejs/pull/829/files#diff-eca12c0a30e25b4b46522ebf89465a03ba72a03f540796c979137931d8f92055R3

I am wondering if we should put the aws-cli to the requirements too like this

> Install the [aws-cli](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html) for e.g. publishing AWS layers.

Reply:
done, please check again